docs/distributed_scenarios.rst

```diff
@@ -0,0 +1,332 @@
+Running distributed scenarios
```

Comment:
I take it from you adding these instructions that this scenario is intended to be used by others right? At some places it feels a bit like: 'I've done this experiment, I hope it also works for you, good luck!'. I think it's really subtle, but you could try rewriting it a bit so it feels more like we expect loads and loads of users :) i.e. that it reads more like: 'This is how you should run this experiment'

Reply:
I added a few suggestions to change the tone a bit, curious what you think of it!

Reply:
"I've done this experiment, I hope it also works for you, good luck!" is a pretty good description of the state this software is in, in my perception. I'll come back to this documentation once I actually have some decent unit test coverage and error handling and rephrase things here and there, but for now that would violate truth in advertising regulations ;-).

Reply:
OK, I think I understand your motivation for writing it this way. I still disagree ;) I'm exaggerating, but the way you see it is that a user-friendly (in the sense of inviting people to use the software instead of scaring away) immediately promises a finished, reliable product. You can also have user-friendly documentation and have a disclaimer somewhere that it is still a prototype. The code quality and documentation of mahiru is much better than some software out there that people use everyday in production.

Reply:
This is my reply also to the other comments, and I think this is a huge improvement over the old code, so feel free to merge!